src/main/java/org/fundacionjala/coding/ariel/movies/Customer.java

```diff
@@ -0,0 +1,78 @@
+package org.fundacionjala.coding.ariel.movies;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * KATA9.
+ * Class Customer
+ */
+public class Customer {
+    private static final int QUANTITY = 50;
+    private final String customer;
+    private List<Rental> rentals;
+    private static final char SLASH = '\n';
+
+
+    /**
+     * @param customer Init the class.
+     */
+    public Customer(String customer) {
+        this.customer = customer;
+        rentals = new ArrayList<>();
+    }
+
+    /**
+     * This method add a rental on the List.
+     *
+     * @param rental is an object of class Rental.
+     */
+    public void addRental(Rental rental) {
+        rentals.add(rental);
+    }
+
+    /**
+     * This method calculates total of rentals movies.
+     *
+     * @return the total of the movies.
+     */
+    public double calculateTotalAmount() {
+        double totalAmount = 0;
+
+        for (Rental rental : rentals) {
+            totalAmount += rental.calculateAmount();
+        }
+        return totalAmount;
+    }
+
+    /**
+     * This method calculates frequent points.
+     *
+     * @return the total frequent points.
+     */
+    public int calculateTotalFrequentPoints() {
+        int totalFrequentRenterPoints = 0;
+        for (Rental rental : rentals) {
+            totalFrequentRenterPoints += rental.calculateFrequentRenterPoints();
+        }
+        return totalFrequentRenterPoints;
+    }
+
+    /**
+     * This method shows the summary of the rentals.
+     *
+     * @return a summary of rentals.
+     */
+    public String statement() {
+        StringBuffer result = new StringBuffer(QUANTITY);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace the synchronized class "StringBuffer" by an unsynchronized one such as "StringBuilder". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1149)